scripts/xsslint_thresholds.json

```diff
@@ -25,7 +25,7 @@
         "python-interpolate-html": 64,
         "python-parse-error": 0,
         "python-requires-html-or-text": 0,
-        "python-wrap-html": 226,
+        "python-wrap-html": 228,
```

Comment:
There’s something fishy with this last change. If we are making a rule change that is moving something from being a violation to being acceptable, then the threshold should be staying the same or going down. Does this just need to be rebased?  Or, could you explain why this is?  Note that the number of violations in your commit is 0 according to the report.